spotbugs/src/main/java/edu/umd/cs/findbugs/ba/AnnotationDatabase.java

```diff
@@ -109,7 +109,7 @@ public boolean anyAnnotations(AnnotationEnum n) {
     public AnnotationEnum getResolvedAnnotation(Object o, boolean getMinimal) {
         if (o instanceof XMethod) {
             XMethod m = (XMethod) o;
-            if (m.getName().startsWith("access$")) {
+            if (m.isAccessMethod()) {
```

Comment:
I took the opportunity to create an `isAccessMethod()` to replace these hardcoded strings